kafka-rest/src/test/java/io/confluent/kafkarest/testing/KafkaRestFixture.java

```diff
@@ -76,6 +76,11 @@ public void before() throws Exception {
     server = application.createServer();
     server.start();
     baseUri = server.getURI();
+    if (certificates != null && baseUri.getScheme().equals("http")) {
+      // If PROXY protocol is enabled, the base URI will come back as http because the first connection
+      // factory is not the SSL factory. If we want SSL, explicitly set the scheme here
+      baseUri = new URI(baseUri.toString().replace("http", "https"));
```

Comment:
nit: We have loaded the JAX-RS classes for Kafka REST, so it might be a tad nicer to use `javax.ws.rs.core.UriBuilder` here:
```
baseUri = UriBuilder.fromUri(baseUri).scheme("https").build();
```